ROADMAP.md

```diff
@@ -4,55 +4,132 @@ This document provides information on OpenEBS development in current and upcomin
+  - https://github.com/openebs/maya/tree/master/cmd/provisioner-localpv
+  - https://github.com/openebs/zfs-localpv
+  - https://github.com/openebs/rawfile-localpv
+- Backlogs
+  - Support for incremental and full Backups for ZFS Local PV
+  - Split the Local Provisioner for hostpath and device from openebs/maya into its own repository
+  - Support for specifying total capacity of all volumes and max capacity per volume that can be provisioned from a given hostpath
+  - Support for specifying multiple hostpaths to be used with Local PV 
+  - Support for specifying node affinity on Local Volumes using custom labels
+  - Add additional integration and end-to-end tests
+  - Helm Charts for ZFS Local PV
+  - Multi-arch builds for all Local PV components  
+
+### Mayastor
+- Project Tracker
+- Source repositories
+  - https://github.com/openebs/Mayastor
+- Backlogs
+  - Support for VolumeSnapshot
+  - Mayastor Replica placement should be topology aware
+  - Mayastor should expose metrics which meet the needs of the SRE persona, to trend review throughput, latency, capacity utilisation and errors
+  - User applications can continue to access volumes when the nexus hosting them fails (e.g. Mayastor container crashes or is otherwise rescheduled, or its host node is lost or disconnected)
+  - It should be possible for Moac (and all other significant control plane components) to be rescheduled within a cluster  
+  - Multi-arch builds for all Mayastor components
+
+### Jiva
+- [Project Tracker](https://github.com/orgs/openebs/projects/1)
+- Source repositories
+  - https://github.com/openebs/jiva
+  - https://github.com/openebs/jiva-operator
+  - https://github.com/openebs/jiva-csi
+- Backlogs
+  - Enhance Jiva Operator functionality to reduce manual steps around launching new replicas when node is completely removed from the cluster
+  - Add additional integration tests to Jiva CSI Driver to move towards beta
+  - Multi-arch builds for Jiva
+  
+### cStor
+- [Project Tracker](https://github.com/orgs/openebs/projects/9)
+- Source repositories
+  - https://github.com/openebs/libcstor
+  - https://github.com/openebs/istgt
+  - https://github.com/openebs/cstor-csi
+  - https://github.com/openebs/cstor-operators
+  - https://github.com/openebs/velero-plugin
+  - https://github.com/openebs/api
+  - https://github.com/openebs/upgrade
+- Backlogs
+  - Multi-arch builds for all cstor containers
+  - Helm chart to install only cStor components
+  - Move the Backup/Restore related API to v1
+  - Move cstor volume and pool metrics code from openebs/maya to openebs/cstor-operators
+  - Additional integration and e2e tests to help move cStor towards stable
+
+### OpenEBS CLI
+- Source repositories
+  - https://github.com/openebs/openebsctl
+- Backlogs
+  - Add the existing functionality available in [mayactl](https://github.com/openebs/maya/tree/master/cmd/mayactl) for volume management to openebsctl
+  - Add Local PV volumes related status checks to openebsctl 
```

Comment:
Should we also add the `Functionality to interface with gRPC layer in NDM` here?

Reply:
This will part of the design, while should be sorted out in OEP or PR. This is another good thread to track this discussion on how to interface NDM gRPC to openebsctl. https://github.com/openebs/openebs/discussions/3140 Once we reach a consensus, we can add that to this backlog.